src/views/Completed.vue

```diff
@@ -332,6 +331,15 @@ export default defineComponent({
             handler: async () => {
               let orderList = JSON.parse(JSON.stringify(this.completedOrders.list))
 
+              // if there are orders with tracking required and label image present
+              const trackingRequiredOrders = orderList.filter((order: any) => this.isTrackingRequiredForAnyShipmentPackage(order))
+              if (trackingRequiredOrders.length) {
+                const orderHasMissingLabelImage = orderList.some((order: any) => order.missingLabelImage)
```

Comment:
I think the terminology here should be around "trackingCode" so that even if label image is missing but user puts tracking code on shipment the variable still makes semantic sense.